python/py-game/Portfile

```diff
@@ -18,9 +18,9 @@ long_description    Pygame is a set of Python modules designed for \
 homepage        http://www.pygame.org/
```

Comment:
Can we make all URLs HTTPS?

Reply:
I realise this is contradictory to what @herbygillot requested, but I'd rather you didn't make unrelated changed to py-game as part of this PR.

Reply:
I don’t need py-game to be a part of this pull request at all now, since 5db97819bb04 independently added a py312 subport to it. All that’s left is cleanup. So I broke the py-game cleanup out into #22256 (including both `port lint` warnings and the https URL), and removed it from this one. Is that better?